### PR TITLE
[TH2-2499] Update dependencies

### PR DESCRIPTION
**Pull request by RaevskiySV**

### CODE REVIEW

File: .github/workflows/dependabot-java-build.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Dependabot build check
 on:
   push:
     branches:
-      - dependabot*
+      - dependabot**
```
Comment (OptimumOpium) [inline .github/workflows/dependabot-java-build.yml:6]: Why did you add an extra `*`?

Comment (RaevskiySV) [reply .github/workflows/dependabot-java-build.yml:6]: Extra `*` is used for proper filtering of branches created by Dependabot.
With single `*` branches are not detectable, since default name is `dependabot/...` and slash symbol is not covered by single `*`.

Comment (OptimumOpium) [reply .github/workflows/dependabot-java-build.yml:6]: Thanks for an explanation

File: src/main/java/com/exactpro/th2/act/ActMain.java [hunk 2]
```diff
@@ -38,6 +39,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ActMain.class);
     private static final String OE_ATTRIBUTE_NAME = "oe";
+    private static final MetricMonitor LIVENESS_MONITOR = registerLiveness("act_liveness");
+    private static final MetricMonitor READINESS_MONITOR = registerReadiness("act_readiness");
```
Comment (OptimumOpium) [inline src/main/java/com/exactpro/th2/act/ActMain.java:43]: Shouldn't we use a common monitor instead? The act does not have extra metrics

`LIVENESS_MONITOR` and `READINESS_MONITOR` from `com.exactpro.th2.common.metrics`

Comment (RaevskiySV) [reply src/main/java/com/exactpro/th2/act/ActMain.java:43]: Actually, preregistered monitors are the same as newly created ones, the only difference is the name. 
If it's better to use existing ones - I will correct the code.
